src/PrestaShopBundle/Service/Database/Upgrade.php

```diff
@@ -38,7 +38,7 @@ public function addDoctrineSchemaUpdate()
         $command = new UpdateSchemaCommand();
         $this->application->add($command);
 
-        $this->addCacheClear();
+        $this->addCacheClear(false);
```

Comment:
Think about someone that will read this, months later: it's totally impossible to figure out what is done here.

And I'm pretty sure future developers will randomly use false or true, hoping it works for the best.